packages/cxl-ui/src/components/cxl-light-card.js

```diff
@@ -0,0 +1,50 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import { LitElement, html } from 'lit';
+import { customElement, property } from 'lit/decorators.js';
+import '@conversionxl/cxl-lumo-styles';
+import cxlLightCardStyles from '../styles/cxl-light-card-css.js';
+
+@customElement('cxl-light-card')
+export class CXLLightCardElement extends LitElement {
+  static get styles() {
+    return [cxlLightCardStyles];
+  }
+
+  @property({ type: String }) id = '';
+
+  @property({ type: String }) theme = 'light-card';
+
+  @property({ type: String }) title = '';
+
+  @property({ type: String }) time = '';
+
+  @property({ type: String }) instructorPrefix = 'By:';
+
+  @property({ type: String }) instructor = '';
+
+  @property({ type: String }) avatar = '';
+
+  @property({ type: Boolean }) hasFooter = false;
+
+  render() {
+    return html`
+      <div class="container">
+        <header class="entry-header">
+          <vaadin-icon icon="cxl:new"></vaadin-icon>
```

Comment:
"New" is used as a property in #294 and #293 and we should make it same here

Reply:
I made the "new" as a property and it made sense to use the same pattern with completed card. So now the "completed" is also a separate property and I removed the themes because these are not necessary anymore.